model/riscv_fdext_control.sail

```diff
@@ -19,7 +19,8 @@ function clause ext_is_CSR_defined 0x003 = haveFExt() | haveZfinx()
 
 function clause ext_read_CSR 0x001 = Some(zero_extend(fcsr[FFLAGS]))
 function clause ext_read_CSR 0x002 = Some(zero_extend(fcsr[FRM]))
-function clause ext_read_CSR 0x003 = Some(zero_extend(fcsr.bits))
+function clause ext_read_CSR 0x003 =
```

Comment:
This one is a bit odd? The line is shorter than the ones above so I'd expect it to not be wrapped.

Reply:
Didn't notice that, fixed

The wrap_check rule doesn't cover all kinds of chunk now(include field), Because I'm not familiar with all the chunk types. Thank you for pointing that out.